website/docs/configuration/index.html.md

```diff
@@ -1,182 +1,51 @@
-[providers](./providers.html) that each define and manage a
-set of resource types. Most providers are associated with a particular cloud or
-on-premises infrastructure service, allowing Terraform to manage infrastructure
-objects within that service.
-
-Terraform doesn't have a concept of platform-independent resource types
-— resources are always tied to a provider, since the features of similar
-resources can vary greatly from provider to provider. But Terraform CLI's shared
-configuration engine ensures that the same language constructs and syntax are
-available across all services and allows resource types from different services
-to be combined as needed.
-
-## Example
-
-The following simple example describes a simple network topology for Amazon Web
-Services, just to give a sense of the overall structure and syntax of the
-Terraform language. Similar configurations can be created for other virtual
-network services, using resource types defined by other providers, and a
-practical network configuration will often contain additional elements not
-shown here.
-
-```hcl
-terraform {
-  required_providers {
-    aws = {
-      source  = "hashicorp/aws"
-      version = "~> 1.0.4"
-    }
-  }
-}
-
-variable "aws_region" {}
-
-variable "base_cidr_block" {
-  description = "A /16 CIDR range definition, such as 10.1.0.0/16, that the VPC will use"
-  default = "10.1.0.0/16"
-}
-
-variable "availability_zones" {
-  description = "A list of availability zones in which to create subnets"
-  type = list(string)
-}
-
-provider "aws" {
-  region = var.aws_region
-}
-
-resource "aws_vpc" "main" {
-  # Referencing the base_cidr_block variable allows the network address
-  # to be changed without modifying the configuration.
-  cidr_block = var.base_cidr_block
-}
-
-resource "aws_subnet" "az" {
-  # Create one subnet for each given availability zone.
-  count = length(var.availability_zones)
-
-  # For each subnet, use one of the specified availability zones.
-  availability_zone = var.availability_zones[count.index]
-
-  # By referencing the aws_vpc.main object, Terraform knows that the subnet
-  # must be created only after the VPC is created.
-  vpc_id = aws_vpc.main.id
-
-  # Built-in functions and operators can be used for simple transformations of
-  # values, such as computing a subnet address. Here we create a /20 prefix for
-  # each subnet, using consecutive addresses for each availability zone,
-  # such as 10.1.16.0/20 .
-  cidr_block = cidrsubnet(aws_vpc.main.cidr_block, 4, count.index+1)
-}
-```
-
-For more information on the configuration elements shown here, use the
-site navigation to explore the Terraform language documentation sub-sections.
-To start, see [_Resource Configuration_](./resources.html).
+_The Terraform language is Terraform's primary user interface._ In every edition
+of Terraform, a configuration written in the Terraform language is always at the
+heart of the workflow.
+
+## What's in This Documentation
```

Comment:
Could we get rid of this heading and just combine the sections above and below? Some of the text seems redundant and not super informative.

Reply:
Hmm, possibly. I'd like to see what Petros has to say before jumping on that, since I asked him to evaluate the on-ramps and through-paths here from a new-ish user's point of view.